find/scripts/extract_download_logs.py

```diff
@@ -74,23 +72,36 @@ def extract_ids_from_message(message: List[List[dict]]) -> List[str]:
 
 def fetch_user_data(user_ids: List[str]) -> dict:
     """
-    Fetch user data from account-store API, using users' azure_ad_subject_id-s
+    Fetch user data from account-store API, using users' account_id-s
     args: user_ids: list of user_ids
     returns: dict of user_id: email_address
     """
+
     responses = {}
-    for user_id in user_ids:
-        try:
-            response = requests.get(
-                f"{Config.ACCOUNT_STORE_API_HOST}/accounts",
-                headers={"Content-Type": "application/json"},
-                params={"azure_ad_subject_id": user_id},
-            )
-            response.raise_for_status()
-            response_data = response.json()
-            responses[user_id] = response_data["email_address"]
-        except requests.exceptions.RequestException as e:
-            print(f"An error occurred for user_id {user_id}: {e}")
+    aws_account_store_api_host = "http://fsd-account-store.dev.pre-award.local:8080"
```

Comment:
Please define this in the download-report manifest: https://github.com/communitiesuk/funding-service-design-post-award-data-store/blob/main/copilot/download-report/manifest.yml as `ACCOUNT_STORE_API_HOST`.